### PR TITLE
Feat : Cart page 

### PR DESCRIPTION
**Pull request by dusariudaykumar**
# Cart Page Static

## cart page contains
- Cart-Card component
- Price Card component 

## Screen shot
scrnli_3_29_2022_7-31-42 PM

### CODE REVIEW

File: src/components/CartCard/CartCard.css [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  margin: 0.7rem;
+}
+.cart-card-title {
+  font-size: 1.2rem;
+}
+
+.cart-remove-item,
+.cart-wishlist-btn {
+  display: inline-block;
+  display: inline-flex;
```
Comment (devdeadviz) [inline src/components/CartCard/CartCard.css:38]: Why you are using two display properties together?

**Review by devdeadviz — APPROVED** [review]
You have written your code in a clean way, and your folder structure is also good. You have taken care of every best practice. Good Work!

File: src/components/CartCard/CartCard.css [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  align-items: center;
+  justify-content: center;
+  border-radius: 0.3rem;
+  outline: none;
+  border: none;
+  gap: 0.4rem;
+  width: 100%;
+  padding: 0.5rem;
+  font-size: 1.1rem;
+  margin: 0.4rem 0;
```
Comment (sharmayatendra) [inline src/components/CartCard/CartCard.css:48]: if value is less than 10px you can use px unit just a suggestion you can skip this

**Review by sharmayatendra — APPROVED** [review]
1- good file structure
2- good that you are importing from a single index file

**Pull request merged**